src/lucky_env.cr

```diff
@@ -23,4 +23,18 @@ module LuckyEnv
       load(file_path)
     end
   end
+
+  def self.name
+    ENV.fetch("LUCKY_ENV", "development")
+  end
+
+  {% for env in %w[development test production] %}
```

Comment:
Can you add `staging` to this? I think that will cover the 99% use case, then later on we can figure out a nice way to make it more configurable at compile-time.

Reply:
@jwoertink I can, but we don't seem to include it by default right now, so I chose not to. However, I just added https://github.com/luckyframework/lucky_env/pull/13/commits/6ad252bde6eb8b38b47685dfa1ff31531af693be which makes it easier to extend `LuckyEnv` with custom environments. So instead of hardcoding a list within `LuckyEnv` itself, we can now just call `LuckyEnv.add_env :staging` to add this environment as well.